### DISCUSSION TITLE
Are postUpgradeTasks run in onboarding PRs?

### DISCUSSION THREAD

**Discussion by flowertwig** [renovatebot/renovate#21071] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

In onboarding PRs, I would like to be able to run postUpgradeTasks, or something similar, in order make changes to renovate.json based on the maven pom file in the repo being onboarded.
The branch name template should serve to detect an onboarding PR.

### What you tried so far.

I've read the postUpgradeTasks documentation and this line(my emphasis) would seem to answer my question in the negative:

> Post-upgrade tasks are commands that are executed by Renovate _after a dependency has been updated_ but before the commit is created.

I've not yet tried this in practice, in the hope that I could a straight answer or an alternative solution to pursue.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
No, they're not run in onboarding PRs
